packages/experimental/RadioGroup/src/Radio/RadioTokens.ts

```diff
@@ -19,12 +19,12 @@ export const defaultRadioTokens: TokenSettings<RadioTokens, Theme> = (t: Theme)
     color: t.colors.neutralForeground3,
     radioOuterCircleSize: 20,
```

Comment:
You should be able to use a global size token for these numbers.

Reply:
Replaced.